svg/src/parser/path.rs

```diff
@@ -6,6 +6,26 @@ use path::math::{Point, point, Vector, vector, Radians};
 use path::builder::SvgBuilder;
 use super::error::ParserError;
 
+/// Builds path object using an SvgBuilder and a list of commands.
+/// Once the path is built you can tessellate it.
+///
+/// The [SvgBuilder](trait.SvgBuilder.html) Adds to [PathBuilder](traits.PathBuilder.html)
+/// the rest of the [SVG path](https://svgwg.org/specs/paths/) commands.
+///
+/// # Examples
+///
+/// ```
+/// # extern crate lyon_svg as svg;
+/// # extern crate lyon_path;
+/// # use lyon_path::default::Path;
+/// # use svg::parser::build_path;
+/// # fn main() {
+/// // Create a simple path.
+/// let commands = &"M 0 0 L 10 0  10 10 L 0 10 z";
+/// let svg_builder = Path::builder().with_svg();
+/// let path = build_path(svg_builder, &commands);
```

Comment:
nittiest nit: not a big deal but you are passing a `&&str` which gets auto-dereffed into a `&str`. you can just pass `commands` without the `&`.
I am a bit pickier than I should but since it is documentation, I'd rather make sure the example doesn't make it look like `build_path` takes a `&&str`

Reply:
Please be as picky as you can ! I know you don't mean to be rude, and I want to improve as much as possible ! :)